docs/csharp/programming-guide/classes-and-structs/codesnippet/CSharp/named-and-optional-arguments_1.cs

```diff
@@ -3,22 +3,27 @@ class NamedExample
         static void Main(string[] args)
         {
             // The method can be called in the normal way, by using positional arguments.
-            Console.WriteLine(CalculateBMI(123, 64));
+            PrintOrderDetails("Foo", 31, true);
```

Comment:
In all locations where you call `PrintOrderDetails`, let's change the order name to a fake name that has some meaning: examples might be "gift order", "inventory replacement", or "recurring order".

Reply:
Makes sense. Updated the code snippet to use fake yet contextual entries.